tools/authors/harvest.php

```diff
@@ -242,7 +237,8 @@ function ensure_digits($digits_or_question_mark) {
 
 // Returns true if the table exists in the current database, false otherwise.
 function table_exists($tableName) {
```

Comment:
You can remove this function and the block at the top of the file that validates these exist. They will always exist since they are part of the base install. This file must have predated how we do table creations.